storage/2020-08-04/blob/blobs/immutability_policy_blob.go

```diff
@@ -0,0 +1,128 @@
+package blobs
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/validation"
+	"github.com/tombuildsstuff/giovanni/storage/internal/endpoints"
+)
+
+type ImmutabilityPolicyBlobInput struct {
+	UntilDate  *string
+	PolicyMode *string
```

Comment:
since both are required - we could make these non-pointers (and PolicyMode a constant)?

```suggestion
	UntilDate  string
	PolicyMode ImmutabilityPolicyMode
```

Reply:
Ah, missed that, thanks, but PolicyMode is optional.